translations/es.ts

```diff
@@ -724,7 +724,7 @@ Disabling chat history now will leave the encrypted history intact (but not usab
         <location filename="../src/chatlog/content/filetransferwidget.cpp" line="311"/>
         <source>Paused</source>
         <comment>file transfer widget</comment>
-        <translation>Pausado</translation>
+        <translation>Pausada</translation>
```

Comment:
it looks strnage to me to paude a feminie thing and then have a masculine thing paused, but if you think it looks better, I can change that one back.